README.md

```diff
@@ -1,369 +1,137 @@
- 1 file changed, 0 insertions(+), 0 deletions(-)
- create mode 100644 new-feature
-```
+## Explain Branching and Committing Changes
 
-Now the branches have been merged and if you `ls`, you'll see the `new-feature` file from the `new-feature` branch in your current working directory that is checked out to master.
+Best practices suggest that any new set of changes related to fixing a bug, creating a feature,
+or even messing around with experimental code in a "sandbox", should be started on a new branch.
+In order to start a new branch, in the terminal type `git branch <branch name>` to create the
+newly defined branch. In the case of a branch relating to isolating a new feature, you could name
+the branch `isolating-new-feature`. The master timeline remains unchanged and clean. This creates
+a new branch which can be seen in the branch list by typing `git branch` in the terminal.
 
-## Working with remote branches with `git fetch` and `git pull`
+## Explain Switching Branches with `git checkout` 
 
-Your local branches can attach to remote branches that live on the internet, generally on GitHub, that your team members might contribute to and you can download locally.
+In order to start making changes on your new branch, you need to "checkout" or move into the
+`isolating-new-feature` timeline or branch, so that git knows that all commits made apply to
+only that unit of work, timeline, or branch. You can move between branches with
+`git checkout <branch name>`. 
 
-Whenever you want to update your local copy with all the branches that might have been added to the GitHub remote, you can type `git fetch -a`.
+**Protip: You can create and checkout a new branch in one command using: `git checkout -b new-branch-name`.
+That will both create the branch `new-branch-name` and move into it by checking it out.**
 
-```
-mission-critical-application $ git fetch -a
-remote: Counting objects: 4, done.
-remote: Compressing objects: 100% (2/2), done.
-remote: Total 4 (delta 3), reused 3 (delta 2), pack-reused 0
-Unpacking objects: 100% (4/4), done.
-From github.com:aviflombaum/mission-critical-application
-   bfe50fc..0ae1da2  master     -> origin/master
- * [new branch]      remote-feature-branch -> origin/remote-feature-branch
-```
+You can always move between branches with `git checkout <branch name>`. If you are currently on
+`isolating-new-feature`, you can move back to master with `git checkout master`. If the last
+branch that you switch from was master, you can also type `git checkout -` in order to move
+back to the previous branch. Then to get back to `isolating-new-feature` you can switch
+it again using `git checkout isolating-new-feature`.
 
-From within `master` (though technically what branch I was in when I typed `git fetch -a` does not matter), I executed `git fetch -a`. The last 3 lines of output are really important, let's take a closer look:
+If you have changes that are not yet committed, when you switch between branches, the untracked
```

Comment:
This is getting git-confusing here. Let's change this to be

"Be sure to make sure that all of your works in progress and new files are committed before your change branches. If you created scratch files you don't want, delete them. If you made changes that you're attached to, make sure to commit them. (new para.) 

The master branch..."